src/ArcxAnalyticsSdk.ts

```diff
@@ -392,6 +381,8 @@ export class ArcxAnalyticsSdk {
       event,
       attributes,
       url: window.location.href,
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      libraryType: getLibraryType(),
```

Comment:
I have a feeling this should be returning a `string` type instead of any, although don't really wanna dive into it as it's not influential 😅

Reply:
It returns a `LibraryType`, I forgot to remove the comment, good catch!